app/src/main/java/com/codepath/apps/critterfinder/fragments/SwipeablePetsFragment.java

```diff
@@ -30,13 +31,15 @@
  * Our tinder-like pet browser
  */
 public class SwipeablePetsFragment extends Fragment implements PetSearch.PetSearchCallbackInterface,
-        SwipeFlingAdapterView.onFlingListener {
+        SwipeFlingAdapterView.onFlingListener,
+        SwipeFlingAdapterView.OnItemClickListener {
```

Comment:
implement an on item click listener on our card adapter view so we can get notified when the user taps on a card.